graph/resolvers/comment.js

```diff
@@ -26,6 +26,7 @@ const Comment = {
 
     query.asset_id = asset_id;
     query.parent_id = id;
+    query.statuses = ['NONE', 'ACCEPTED'];
```

Comment:
Same as above.

Reply:
Reply query currently does not allow specifying `statuses`.